code/datums/components/tackle.dm

```diff
@@ -233,15 +262,111 @@
+ * * 21 to 49: Our tackler is knocked down, suffers stamina damage, and is off-balance. Ouch.
+ * * 50 to inf: Our tackler suffers a catastrophic failure, receiving significant stamina damage, a concussion, and is paralyzed for 3 seconds. Oh, and they're off-balance for a LONG time.
+*/
+
+/datum/component/tackler/proc/negative_outcome(mob/living/carbon/user, mob/living/carbon/target, roll = -1, tackle_word = "tackle")
+	var/potential_roll_outcome = (roll * -10)
+
+	if(ishuman(user))
+		var/mob/living/carbon/human/human_sacker = target
+		var/attacker_armor = human_sacker.run_armor_check(BODY_ZONE_CHEST, MELEE)
+		potential_roll_outcome *= ((100 - attacker_armor) /100)
+	else
+		potential_roll_outcome *= 0.9
+
+	var/actual_roll = rand(1, potential_roll_outcome)
+
+	var/mob/living/carbon/human/human_target = target
+	var/mob/living/carbon/human/human_sacker = user
+
+	switch(actual_roll)
+
+		if(-INFINITY to 0) //I don't want to know how this has happened, okay?
+			neutral_outcome(user, target, roll, tackle_word) //Default to neutral
+
+		if(1 to 20) // It's not completely terrible! But you are somewhat vulernable for doing it.
+			user.visible_message(span_warning("[user] lands a weak [tackle_word] on [target], briefly knocking [target.p_them()] off-balance!"), span_userdanger("You land a weak [tackle_word] on [target], briefly knocking [target.p_them()] off-balance!"), ignored_mobs = target)
+			to_chat(target, span_userdanger("[user] lands a weak [tackle_word] on you, briefly knocking you off-balance!"))
+
+			user.Knockdown(1 SECONDS)
+			target.apply_damage(20, STAMINA)
+			if(ishuman(target) && !human_target.has_movespeed_modifier(/datum/movespeed_modifier/shove))
+				human_target.add_movespeed_modifier(/datum/movespeed_modifier/shove)
+				addtimer(CALLBACK(human_target, TYPE_PROC_REF(/mob/living/carbon, clear_shove_slowdown)), SHOVE_SLOWDOWN_LENGTH * 2)
+
+			if(ishuman(user) && !human_sacker.has_movespeed_modifier(/datum/movespeed_modifier/shove))
+				human_sacker.add_movespeed_modifier(/datum/movespeed_modifier/shove)
+				addtimer(CALLBACK(human_sacker, TYPE_PROC_REF(/mob/living/carbon, clear_shove_slowdown)), SHOVE_SLOWDOWN_LENGTH * 2)
+
+		if(21 to 49) // oughe
+			user.visible_message(span_warning("[user] lands a dreadful [tackle_word] on [target], briefly knocking [user.p_them()] to the ground!"), span_userdanger("You land a dreadful [tackle_word] on [target], briefly knocking you to the ground!"), ignored_mobs = target)
+			to_chat(target, span_userdanger("[user] lands a dreadful [tackle_word] on you, briefly knocking [user.p_them()] to the ground!"))
+
+			user.Knockdown(3 SECONDS)
+			user.apply_damage(40, STAMINA)
+			if(ishuman(user) && !human_sacker.has_movespeed_modifier(/datum/movespeed_modifier/shove))
+				human_sacker.add_movespeed_modifier(/datum/movespeed_modifier/shove)
+				addtimer(CALLBACK(human_sacker, TYPE_PROC_REF(/mob/living/carbon, clear_shove_slowdown)), SHOVE_SLOWDOWN_LENGTH * 2)
```

Comment:
Why are we only doing this to humans anyway? Any mob can have a movespeed modifier
Either way you can move the type checking into the status effect too if you want to keep it and it will handle not reapplying by itself

Reply:
Give me an opportunity to make this a status effect and we won't have to worry about this callback whatsoever.